### PR TITLE
Fixed: Multiple genre's is not being handled correctly

### PR DESCRIPTION
**Pull request by Starwer**
This is a fix for #111:

- fixed: use of semicolon as list-separator for the `TCON` (Genres) prior to ID3 v2.4 was not supported;
-  added test for ID3 v2.3 Genres Tag

### CODE REVIEW

**Comment by Starwer** [comment]
Please hold integration. There is still work to do on that feature branch.

**Comment by Starwer** [comment]
Branch is now fully ready for integration.

File: src/TagLib/Ogg/XiphComment.cs [hunk 1]
```diff
@@ -52,12 +52,6 @@ public class XiphComment : TagLib.Tag, IEnumerable<string>
 		/// </summary>
 		private string vendor_id;
 
-		/// <summary>
-		///    Contains the field identifier to use for <see
-		///    cref="Comment" />.
-		/// </summary>
-		private string comment_field = "DESCRIPTION";
-
```
Comment (decriptor) [inline src/TagLib/Ogg/XiphComment.cs]: Why get rid of this?

Comment (Starwer) [reply src/TagLib/Ogg/XiphComment.cs]: The compiler complains (warning) that this is not used anywhere. This has anyway been removed in the meantime from base (integration #109) so this change is re-done (can be ignored, same as base).

**Pull request merged**